var/spack/repos/builtin/packages/py-bakta/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyBakta(PythonPackage):
+    """Bakta: rapid & standardized annotation
+    of bacterial genomes, MAGs & plasmids"""
+
+    homepage = "https://github.com/oschwengers/bakta"
+    pypi = "bakta/bakta-1.5.1.tar.gz"
+
+    maintainers = ["oschwengers"]
```

Comment:
@oschwengers Are you okay with being added as a maintainer of this package?

Reply:
Hi! Yes, I'm OK with that. I have no experience with Spack so far, but I'll jump in if I can help with any tool-related questions/issues.